doc/source/train/hf_trainers.rst

```diff
@@ -0,0 +1,51 @@
+Hugging Face
```

Comment:
cc @matthewdeng, moving this out of AIR and putting it into Train for now; you will probably get rid of this with the content refactor?